Segmentation and intensity feature extraction (#6)
AllenCell/PairUPWorkshop: segmentation_and_intensity/README.md

```
python feature_extraction.py --input_seg_dir segmentation_dir --input_raw_timelpase_dir aligned_data_dir --feature_manifest_dir feature_output_manifest_dir
```

## Data visualizatioin using colorizer

To visualize these results, we utilize [Timelapse Feature Explorer (TFE)](https://github.com/allen-cell-animated/timelapse-colorizer)

We first convert our features and segmentations into a format that colorizer can read:

```
python colorizer_conversion.py --parent_input_csvs_dir output_dir_of_feature_manifest_dir --output_dir colorizer_output_directory

python make_collection.py --colorizer_input_dir colorizer_input_directory_for_converted_data # This is if you have multiple movies that you want to view as a collection
```

This produces a collection file that you can then upload and view on colorizer

To view your data on the viewer:

```
# starts a local server 
python run_local_server.py 8080

# On your browser, go to http://localhost:8080

Copy the address of your collection file of interest and click the timelapse-feature-explorer directory to open the viewer and upload that address in the load data tab

```

> ## Data visualizatioin using web volume viewer

Reviewer: ```suggestion
## Data visualization using web volume viewer
```

> We can also use the [Web-volume viewer](https://github.com/allen-cell-animated/volume-viewer) to visualize the results. A neccesary step here is converting the data into the OME-Zarr format

Reviewer: ```suggestion
We can also use the [3D Volume Viewer](https://github.com/allen-cell-animated/volume-viewer) to visualize the results. A necessary step here is converting the data into the OME-Zarr format
```

```
# Converts timelapse data into the OME-Zarr standard
python convert_to_zarr.py --input_dir input_dir --output_dir output_converted_movies_path

#  similarly start a local server 
python run_local_server.py 8080

Copy the address of the Zarr file of interest and click the 3d-volume-viewer directory to open the viewer and upload that address in the load data tab

> ```

Reviewer: ````suggestion
```

Copy the address of the Zarr file of interest and click the 3d-volume-viewer directory to open the viewer and upload that address in the load data tab
````

## Training a custom cellpose model(Optional)

Cellpose has many standard models that you can apply. This is often a great place to start to get a first pass segmentation. 

> However, if these are not generalizable, you can also train a custom model. We recomend using the cellpose GUI for creating annotations for a subset of data and exporting the annotations as png masks.

Reviewer: Nit: Only these sentences have punctuation?
